client/login/wp-login/index.jsx

```diff
@@ -434,42 +433,29 @@ export class Login extends Component {
 		}
 
 		if ( ( isWoo || isBlazePro ) && isLoginView ) {
-			return (
-				<>
-					<LoginFooter lostPasswordLink={ this.getLostPasswordLink() } shouldRenderTos />
-				</>
```

Comment:
Here I used the opportunity to remove some pointless `<>` fragments with a single child element.